server/package.json

```diff
@@ -11,7 +11,8 @@
     "build": "tsc -p tsconfig.build.json",
     "lint": "eslint",
     "typecheck": "tsc --noEmit",
-    "test": "jest"
+    "test": "jest",
+    "up": "taze -I"
```

Comment:
Was macht den das mit -I Flag? Habe das in der Doku nicht gefunden.
Man könnte das auch eine Ebene höher hängen und dann mit -R direkt Server und Client scannen.

Reply:
`yarn taze --help
  -I, --interactive   interactive mode                [boolean] [default: false]`